src/daemon/absorber_test.cc

```diff
@@ -8,15 +8,24 @@ namespace daemon {
 
 namespace {
 
+static const ui32 kEmptyEmitterVersion = 0u;
+static const ui32 kBadEmitterVersion = 1u;
```

Comment:
Anyway, it's better not to use a module-wide constants outside of test bodies - just for couple of them.